internal/bminventory/inventory_test.go

```diff
@@ -14038,8 +14246,14 @@ var _ = Describe("GetInfraEnvPresignedFileURL", func() {
 			Expect(u.Query().Get("image_token")).To(Equal(""))
 			Expect(u.Query().Get("api_key")).To(Equal(""))
 			Expect(u.Query().Get("file_name")).To(Equal("discovery.ign"))
+			Expect(u.Query().Get("boot_control")).To(BeEmpty())
```

Comment:
Can we instead test that there is no `boot_control` key rather than the value being empty?
I just want to make sure the discovery ignition URL doesn't contain that parameter, it would look pretty strange.

Reply:
Tested